### DISCUSSION TITLE
yank too slow

### DISCUSSION THREAD

**Discussion by yinghaoyu** [ayamir/nvimdots#1060] [Q&A] [answered]
In `WSL-ubuntu20.04`, yank one line costing about 5 seconds, which is too slow. Does anyone same like this ?

**Comment by charliie-dev** [comment]


**Reply by yinghaoyu** [reply]
what does `setup` means?
```bash
$which win32yank.exe
/usr/local/bin/win32yank.exe
```

**Comment by ayamir** [comment]
add `win32yank.exe` to the PATH of Windows.

**Reply by yinghaoyu** [reply]
Should I install  `win32yank.exe` both in WSL and windows?

**Reply by yinghaoyu** [reply]
I just install `win32yank.exe` in WSL with:
```bash
curl -sLo/tmp/win32yank.zip https://github.com/equalsraf/win32yank/releases/download/v0.0.4/win32yank-x64.zip
unzip -p /tmp/win32yank.zip win32yank.exe > /tmp/win32yank.exe
chmod +x /tmp/win32yank.exe
sudo mv /tmp/win32yank.exe /usr/local/bin/
```
from [here](https://github.com/neovim/neovim/wiki/FAQ#old-instructions).

**Reply by charliie-dev** [reply]
WSL and Windows share the same file system. u should be able to find `win32yank.exe` in Windows, then add it to Windows' `PATH`.

**Reply by yinghaoyu** [reply]
```bash
$which win32yank.exe
/usr/local/bin/win32yank.exe
```
I had added `\\wsl.localhost\Ubuntu\usr\local\bin` to Windows's `Path `, but it seems not working.

**Reply by ayamir** [reply, marked as answer]
Download it on Windows and add it to `PATH` of Windows.

**Reply by yinghaoyu** [reply]
Thanks. It's the only right way to solve this problem.

**Comment by charliie-dev** [comment]


**Reply by yinghaoyu** [reply]
```txt
Clipboard (optional) ~
- OK Clipboard tool found: win32yank-wsl
```

**Reply by charliie-dev** [reply]
hmmm, no idea how to fix it.
maybe try upgrade nvim or your system?
